specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -112,6 +112,24 @@ Explain how does the system handle API violations.
 
 
 ## Detailed design
+### VDK Notebook plugin
+ This VDK plugin will provide the functionality to run Jobs which will retrieve Job steps from notebook files instead of .py and .sql files. This plugin can be used alone without the JupyterLab extension.
+
+### VDK JupyterLab extension
+ This extension will be both front-end and server side extension for JupyterLab.
+ The front-end side will be introducing the graphical elements such as menus, buttons, etc. and will be responsible with sending
```

Comment:
I missed saying that since it can be done at follow-up PR when you have better idea about these things

But we need to be very explicit here. 

What are the exact graphical elements that would be introduced. Having mockup picture would be a huge bonus . 

What are the exact request that would be sent and what commands would be supported? 

Also there's a lot of assumption about what the reader knows and not everyone reading this would know how VDK works and what is file step plugin and etc , so you need either describe or better link to existing documentation that explain concepts you are using.